contracts/RiskManagerV1.sol

```diff
@@ -18,15 +19,23 @@ interface IDeposit {
     function currentState() external view returns (uint256);
 
     function lotSizeTbtc() external view returns (uint256);
+
+    function collateralizationPercentage() external view returns (uint256);
 }
 
 /// @title RiskManagerV1 for tBTCv1
-contract RiskManagerV1 is Auctioneer {
+contract RiskManagerV1 is Auctioneer, Ownable {
     using SafeERC20 for IERC20;
     using SafeMath for uint256;
 
     uint256 public constant DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE = 10;
     uint256 public constant DEPOSIT_LIQUIDATED_STATE = 11;
+    // Auction will not be opened if the deposit collateralization level does not
+    // fall below this collateralization threshold.
+    // Risk manager should open a coverage pool auction for only those deposits
+    // that nobody else is willing to purchase. The default value can be updated
+    // by the governance at any moment.
+    uint256 public collateralizationThreshold = 101; // percent
```

Comment:
That's the crux. Let's create a sheet and try to model different options. This value can be updated by the governance by we need to help them analyze and understand how does it work and assign good default value. Two cases are especially interesting:
- what if the gas price is so high that although taking deposit is theoretically profitable, no one wants to do that?
- what if the collateralization level goes up quickly after the risk manager decides to open an auction?

https://docs.keep.network/tbtc/#_undercollateralization

Reply:
> what if the gas price is so high that although taking deposit is theoretically profitable, no one wants to do that?

Here's a [sheet](https://docs.google.com/spreadsheets/d/18tDejlULfTN4RG_xgfpcG-K5G8aOwhM6kjlQnPiC0Z4/edit?usp=sharing) based on today's BTC/ETH ratio `0.078`. If my understanding is right, then threshold `101%` is not profitable when the gas price is between 875 - 900. Everything below this gas price should be profitable to purchase a deposit outside the coverage pool.

> what if the collateralization level goes up quickly after the risk manager decides to open an auction?

I'd need some help on that one. If collateralization level was low enough to open an auction in coverage pool and then it went up above the threshold, I'm not sure what should happen in this case.  We have a function to early close an auction, but not sure if this is what we want to trigger?

Reply:
In the sheet above, I've considered two scenarios based on 1 BTC as a deposit and the historic ETH/BTC ratios from May '20 - May '21. In May '20 it was `~0.03`, now it is `~0.075`. If we consider fluctuations in % for the average gas price of `125 gwei` then it varies between `100-101`.  At `100%` collateralization is already not profitable to buy, but at `101%` still makes sense to buy outside the coverage pool. So maybe we should change the default value to `100`, wdyt @pdyraga ?

Reply:
I'd recommend opening an issue with those findings plus adding a backlog item to find the correct initial value so that we do not have to block this PR and having this discussion there. WDYT?

Reply:
Sure. [issue](https://github.com/keep-network/coverage-pools/issues/64) & backlog item was also added.